tools/iree-e2e-matmul-test.c

```diff
@@ -276,34 +324,44 @@ static iree_status_t get_matmul_sizes(
   return iree_ok_status();
 }
 
-// Helper for reference_matmul_element. f32 case.
-static void reference_matmul_element_f32(
-    iree_hal_dim_t m_size, iree_hal_dim_t k_size, iree_hal_dim_t n_size,
-    iree_hal_element_type_t lhs_type, iree_hal_element_type_t rhs_type,
-    float* lhs_data, float* rhs_data, float* acc_data, float* result_data,
-    iree_hal_dim_t m, iree_hal_dim_t n) {
-  float acc = acc_data[n + m * n_size];
-  for (iree_hal_dim_t k = 0; k < k_size; ++k) {
-    float lhs_value = lhs_data[k + m * k_size];
-    float rhs_value = rhs_data[n + k * n_size];
-    acc += lhs_value * rhs_value;
+#define IREE_TRACE_REPLAY_REFERENCE_GEMM(LHSTYPE, RHSTYPE, RESTYPE, ACCTYPE)  \
```

Comment:
we use the term "matmul" for matrix multiplication.  "GEMM" refers specifically to the standard BLAS interface, which is not what we are talking about here.